tests/integration/golang/aim/metric/search_test.go

```diff
@@ -5920,14 +5927,22 @@ func (s *SearchMetricsTestSuite) Test_Ok() {
 						IsNan:     false,
 						RunID:     run.ID,
 						LastIter:  int64(decodedData[itersKey].([]float64)[0]),
+						Context:   models.Context{Json: datatypes.JSON(`{}`)},
+						ContextID: 1,
 					}
 					decodedMetrics = append(decodedMetrics, &m)
 					metricCount++
 				}
 			}
 
 			// Check if the received metrics match the expected ones
-			s.Equal(tt.metrics, decodedMetrics)
+			s.Equal(len(tt.metrics), len(decodedMetrics))
+			for i, metric := range tt.metrics {
+				// TODO when encoding.Decoder handles the context object, we can include it in the comparison
```

Comment:
But, when will we TODO it? Should we consider doing it now?

Reply:
created an issue https://github.com/G-Research/fasttrackml/issues/777